staging/src/k8s.io/client-go/tools/reference/ref.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ref
+package reference
```

Comment:
Yes. My understanding is that there will be k8s.io/util for non-kube things, and k8s.io/pkg that can import kube types. So how about this reference package to k8s.io/client-go/pkg for now, and move it to k8s.io/pkg in the future?